src/components/buttons/User.vue

```diff
@@ -4,14 +4,14 @@
       <div>{{ nameLetters }}</div>
       <span>{{ $t('buttons.user') }}</span>
     </div>
-    <div :class="{'fade-on': userToggled}" class="speech-bubble fade">
-        {{$t('header.signedAs')}}&nbsp;
+    <div :class="{'fade-on': userToggled, 'rtl': direction == 'rtl', 'ltr': direction == 'ltr'}" class="speech-bubble fade">
```

Comment:
why not use class binding? as used in fade-on

Reply:
@omerzamir  This style of binding gets a Boolean value (as with userToggled which is a boolean) and I didn’t find another way of mixing the existing fade-on class with the added conditional rtl/ltr.

Reply:
Take a look in [here](https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax).
The whole class object can be computed, it may help.